bindings/node/src/mongocrypt.cc

```diff
@@ -589,10 +577,7 @@ Value MongoCrypt::MakeExplicitEncryptionContext(const CallbackInfo& info) {
 
         if (options.Has("queryType")) {
             std::string query_type_str = options.Get("queryType").ToString();
-            mongocrypt_query_type_t query_type = strToEnumValue<mongocrypt_query_type_t>(
-                Env(), query_type_str, "queryType",
-                {{"Equality", MONGOCRYPT_QUERY_TYPE_EQUALITY}});
-            if (!mongocrypt_ctx_setopt_query_type(context.get(), query_type)) {
+            if (!mongocrypt_ctx_setopt_query_type(context.get(), query_type_str.data(), -1)) {
```

Comment:
I'd prefer not to remove len from setopt_algorithm since that would be a breaking change to a stable api (thus would require a libmongocrypt 2.0 release). Removing it from setopt_query_type sounds fine though since it's still in beta.

Reply:
@vector-of-bool Yeah, I wasn’t concerned about missing NUL at the end, I was thinking about NUL in the middle (e.g. `equality\0asdf` would now also be accepted here).